iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/ImportConfig.java

```diff
@@ -55,6 +51,25 @@ public ImportConfig(List<String> postProcessors) {
         return postProcessors;
     }
 
+    public ImportConfig setPostProcessors(List<String> postProcessors) {
```

Comment:
Please add a comment to explian why use this instead of `new ImportConfig(list)`
Check null as it is a public method

Reply:
I added a comment. Is that comprehensive enough?